sdk/integration-tests/test-utils.tsx

```diff
@@ -36,24 +55,131 @@ import {
   useConnect,
 } from 'wagmi'
 
-const mockConnector = mock({ accounts: [testAccount.address] as const })
+export const devnetApiUrl = 'http://localhost:26661/api/v1'
 
-// biome-ignore lint/suspicious/noExplicitAny: test file
-export function testConnector(config: any) {
-  const connector = mockConnector(config)
-  connector.getClient = async ({ chainId } = {}) => {
-    if (chainId === mockL1Id) {
-      return mockL1Client
-    }
-    const chain = chainId ? mockChains[chainId] : mockL1Chain
-    if (chain == null) {
-      throw new Error(`Unsupported chain: ${chainId}`)
+const txHashRegexp = /^0x[0-9a-f]{64}$/
+
+type WaitForInboxOrderFilledParams = GetOrderParameters & {
+  pollingInterval?: number
+  timeout?: number
+}
+
+function waitForInboxOrderFilled(
+  params: WaitForInboxOrderFilledParams,
+): Promise<void> {
+  const { pollingInterval, timeout, ...getOrderParams } = params
+  return new Promise<void>((resolve, reject) => {
+    const timeoutId = setTimeout(() => {
+      clearInterval(pollId)
+      reject(new Error('Timeout waiting for order to be filled on inbox'))
+    }, timeout ?? 60_000)
+    const pollId = setInterval(async () => {
+      const order = await getOrder(getOrderParams)
+      const status = parseInboxStatus({ order })
+      if (status === 'filled') {
+        clearInterval(pollId)
+        clearTimeout(timeoutId)
+        resolve()
+      }
+    }, pollingInterval ?? getOrderParams.client.pollingInterval)
+  })
+}
+
+type WaitForOutboxOrderFilledParams = DidFillParams & {
+  pollingInterval?: number
+  timeout?: number
+}
+
+function waitForOutboxOrderFilled(
+  params: WaitForOutboxOrderFilledParams,
+): Promise<void> {
+  const { pollingInterval, timeout, ...didFillParams } = params
+  return new Promise<void>((resolve, reject) => {
+    const timeoutId = setTimeout(() => {
+      clearInterval(pollId)
+      reject(new Error('Timeout waiting for order to be filled on outbox'))
+    }, timeout ?? 60_000)
+    const pollId = setInterval(async () => {
+      const isFilled = await didFillOutbox(didFillParams)
```

Comment:
Not a comment about tests, but users do not care about the "outbox" part of `didFillOutbox`. I suggest we strip it. Just `didFill(...)`.